services/web/server/tests/unit/with_dbs/10/test_resource_manager.py

```diff
@@ -497,6 +512,9 @@ async def test_interactive_services_removed_per_project(
     mocked_director_api["stop_service"].reset_mock()
 
 
+@pytest.mark.xfail(
+    reason="it is currently not permitted to open the same project from 2 different tabs"
+)
```

Comment:
was this permitted before? If yes, are we disabling it? In that case I would change the test to see that it properly fails to open from 2 different tabs

Reply:
to answer your questions, yes it was doable from the backend side but prohibited from the frontend.
so now it's in phase.
that marking as xfail actually means that it checks that the test fails.